pypsa_to_IPCC.py

```diff
@@ -9,26 +9,28 @@
 import pandas as pd
 
 #original IPCC file, official template
-path = "global_sectoral/global_sectoral/IPCC_AR6_WG3_Global_sectoral_Pathways_scenario_template_v3.1_20221027.xlsx"
+template_path = "global_sectoral/IPCC_AR6_WG3_Global_sectoral_Pathways_scenario_template_v3.1_20221027.xlsx"
 #use the official template
 
+# Metadata
 model = "PyPSA-Eur-Sec 0.0.2"
+model_version = "0.0.6"
+literature_reference = "Pedersen, T. T., Gøtske, E. K., Dvorak, A., Andresen, G. B., & Victoria, M. (2022). Long-term implications of reduced gas imports on the decarbonization of the European energy system. Joule, 6(7), 1566-1580."
+climate_target = 21 # CO2 budget
+
 wind_split = ['DE', 'ES', 'FI', 'FR', 'GB', 'IT', 'NO', 'PL', 'RO', 'SE']
-scenario_short_name={'Early and Steady':'Base',
-                     'Late and Rapid':'Base',
-                     'District heating expansion':'w_DH_exp',
-                     'Space heat savings due to building renovation': 'w_Retro',
-                     'Transmission expansion after 2030':'w_Tran_exp',
-                     'Including road and rail transport':'w_EV_exp'}
-scenarios =['Early and Steady',
-#            'Late and Rapid',
-#            'District heating expansion',
-#            'Space heat savings due to building renovation',
-#            'Transmission expansion after 2030',
-#            'Including road and rail transport'
-            ]
+
+scenarios={'Base_1.5':'postnetworks/elec_s370_37m_lv1.0__3H-T-H-B-I-solar+p3-dist1-cb25.7ex0_',
+            'Gaslimit_1.5':'postnetworks/elec_s370_37m_lv1.0__3H-T-H-B-I-solar+p3-dist1-cb25.7ex0-gasconstrained_',
+            'Base_2':'postnetworks/elec_s370_37m_lv1.0__3H-T-H-B-I-solar+p3-dist1-cb73.9ex0_',
+            'Gaslimit_2': 'postnetworks/elec_s370_37m_lv1.0__3H-T-H-B-I-solar+p3-dist1-cb73.9ex0-gasconstrained_',
+                     }
+
+
+output_folder = 'results/'
 
 years = [2020, 2025, 2030, 2035, 2040, 2045, 2050]
+
 countries = ['AT','BE','BG','CH','CZ','DE','DK','EE','ES','FI','FR','GB','GR','HR',
```

Comment:
can we read the countries from the network file? so that the script is working for networks that do not contain all the countries? If someone has modelled Germany it should also be able to produce a IAMC file with data from that country